.github/workflows/checks-quick.yml

```diff
@@ -35,6 +35,21 @@ jobs:
         with:
           app-id: ${{ secrets.WORKFLOW_STOPPER_RUNNER_APP_ID }}
           app-key: ${{ secrets.WORKFLOW_STOPPER_RUNNER_APP_KEY }}
+  prdoc:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - uses: actions/checkout@6d193bf28034eafb982f37bd894289fe649468fc # v4.1.7
+      - name: prdoc check
```

Comment:
Why isn't that just part of the `CheckPrdoc` job?

Reply:
I thought it would be similar to `fmt`.

But I refactored and put inside the `CheckPrdoc`: basically it was ignoring errors when `R0-silent`, I make the check mandatory: https://github.com/paritytech/polkadot-sdk/pull/7543/commits/501329508746d555f60a428187c0fcfb9c26fdb8